### PR TITLE
Notification improvements

### PR DESCRIPTION
**Pull request by vladjerca**
1. Moved position to top-right (settings icon is no longer there, so this way the toastr's do not interfere with the application in any way).
2. I've changed the remove song toastr to a warning.
3. I've changed the color scheme using colors from [https://flatuicolors.com/](url), and also removed rounded edges. (the entire interface is based on strong defined corners, the toastr's just didn't belong).

### CODE REVIEW

**Comment by Ajeey** [comment]
A screenshot of previous notification and the new one would be helpful.

**Comment by Pitros** [comment]
It's slightly better now, but still not perfect - now you can't access window action buttons on Windows. @Ajeey 
Before:
image
After:
image

File: app/public/js/common/favoriteSongDirective.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ app.directive('favoriteSong', function(
                     SCapiService.deleteFavorite(userId, songId)
                         .then(function(status) {
                             if ( typeof status == "object" ) {
-                                notificationFactory.success("Song removed from likes!");
+                                notificationFactory.warning("Song removed from likes!");
```
Comment (Pitros) [inline app/public/js/common/favoriteSongDirective.js]: There's no `warning` methog in `notificationFactory`... it should be `noticationFactory.warn()`.

**Comment by vladjerca** [comment]
@Pitros Thanks for the screenshots, I was in a bit of a hurry when I made the changes, didn't take the time to take the screenshots. Hope you guys like it as much as I do.

**Comment by Ajeey** [comment]
I believe if we reduce the size of the image/font-image in the notification toast, it would look more cleaner and compact.

**Comment by weblancaster** [comment]
What were the changes you did to `toastr.min.css` and `angular-toastr.min.css` @vladjerca ? is it just updated to new version of them or you actually updated the code?

if you updated the code (`toastr.min.css` and `angular-toastr.min.css`) can you revert to what was before please and instead just overwrite the css on `_default.scss`

**Comment by vladjerca** [comment]
I used the .less files provided in the official github repositories for each to make the modifications and I just compiled/minified them to replace the old ones.

I can just take the diffs and add them to '_default.scss' in order to override the original classes.

I'll try to tidy up both of the pull requests in the by the end of the week. (some personal matters got in the way)

**Comment by weblancaster** [comment]
I think the changes are good so we just need to revert `toastr.min.css` and `angular-toastr.min.css` back to how it was and override them original on `_default.scss`

**Comment by vladjerca** [comment]
The commit has been updated.

**Comment by weblancaster** [comment]
@vladjerca thanks but when I said to override I wasn't thinking about adding important all over the place..

What you can do is something like this.. override by selector hierarchy 

```css
#app {
// here the toast style without important
}
```

**Comment by vladjerca** [comment]
@weblancaster Yeah, my bad on that one. I've updated the scss file.

**Comment by weblancaster** [comment]
This how looks like (not as your screenshot).. what you want to do is use 'body' instead of `#app` that's why I put the comment to use the selector that would add the hierarchy.

screen shot 2016-02-04 at 11 11 09 am

**Comment by vladjerca** [comment]
@weblancaster Updated, sorry for all the mishaps.

**Pull request merged**